File: src/test/java/com/ject/studytrip/BaseUnitTest.java
package com.ject.studytrip;

import com.fasterxml.jackson.databind.ObjectMapper;
import org.junit.jupiter.api.extension.ExtendWith;
import org.mockito.junit.jupiter.MockitoExtension;

@ExtendWith(MockitoExtension.class)
public abstract class BaseUnitTest {

    protected final ObjectMapper objectMapper = new ObjectMapper();
}

File: src/test/kotlin/com/ject/studytrip/BaseUnitTest.kt
package com.ject.studytrip

import com.fasterxml.jackson.databind.ObjectMapper
import org.junit.jupiter.api.extension.ExtendWith
import org.mockito.junit.jupiter.MockitoExtension

@ExtendWith(MockitoExtension::class)
abstract class BaseUnitTest {
    protected val objectMapper: ObjectMapper = ObjectMapper()
}
